jobs/uaa/templates/config/uaa.yml.erb

```diff
@@ -860,6 +860,9 @@
     add_value(params, ca_certs, 'uaa', 'ca_certs')
   end
 
+  # content security policy (csp) settings
+  add_value(params, p('uaa.csp.script-src'), 'csp', 'script-src') if p_opt('uaa.csp.script-src')
```

Comment:
Could we have a template test that validates this line? It doesn't look to me like this is tested.

Reply:
yes a test would help to ensure syntax